v3/src/components/graph/hooks/use-plot.ts

```diff
@@ -98,7 +98,7 @@ export const usePlotResponders = (props: IPlotResponderProps) => {
       () => GraphAttrRoles.map((aRole) => dataConfiguration?.attributeID(aRole)),
       () => {
         // if plot is not univariate and the attribute type changes, we need to update the pointConfig
-        if (graphModel?.plotType !== "dotPlot") {
+        if (graphModel?.plotType !== "dotPlot" && !graphModel?.pointsFusedIntoBars) {
```

Comment:
This is a bug fix. Without this change, points fused into bars would automatically unfuse back to points if you added a categorical split to the graph.